vault/README.md

```diff
@@ -12,60 +12,64 @@ The Vault check is included in the [Datadog Agent][2] package.
 
 #### Prerequisites
 
+This OpenMetrics-based integration has a latest version (`use_openmetrics`: true) and a legacy version (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. Versions 3.4.0 and later of this check use [OpenMetrics][21] for metric collection, which requires Python 3. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][22].
```

Comment:
I feel like the word "version" gets a bit overloaded in this paragraph, and the `use OpenMetrics for metric collection` actually means using *latest*, which is unclear from that wording.

I've given it some thought and I think something like the following would add clarity:

```suggestion
Starting with version 3.4.0, this OpenMetrics-based integration has a *latest* mode (`use_openmetrics`: true) and a *legacy* mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the *latest* mode. Note that *latest* mode requires Python 3. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][22].
```

I'm still not sure about the mention to Python 3, though, since it looks like most of the other integrations don't mention it.

Let me know what you think.